app/src/main/java/com/weatherxm/data/services/CacheService.kt

```diff
@@ -307,6 +308,14 @@ class CacheService(
         return preferences.getInt(KEY_DEVICES_OWN, 0)
     }
 
+    fun getLastDismissedSurveyId(): String? {
+        return preferences.getString(KEY_DISMISSED_SURVEY_ID, null)
+    }
+
+    fun setLastDismissedSurveyId(surveyId: String) {
+        preferences.edit().putString(KEY_DISMISSED_SURVEY_ID, surveyId).apply()
+    }
```

Comment:
I don't know if it's better to keep a list of the dismissed ids instead of only the last one

Reply:
We had discussed it on the refinement Panteli and decided to keep only the last one so the others can be reused in future iterations of the same survey.